### PR TITLE
Update getting-started-on-the-devops-platform.json

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Added GH Enterprise document

### CODE REVIEW

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Review by mitovskaol — APPROVED** [review]

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

File: web/topicRegistry/getting-started-on-the-devops-platform.json [hunk 1]
```diff
@@ -104,7 +104,8 @@
             "docs/OCP/OCPStorageServices.md",
             "docs/OCP/OCP4-Storage-backup.md",
             "docs/OCP/OCP4ServiceOverview.md",
-            "docs/HowTo/UsefulProTips.md"
+            "docs/HowTo/UsefulProTips.md",
+            "docs/HowTo/RequestGithubEnterpriseUserLicense.md"
```
Comment (patricksimonian) [inline web/topicRegistry/getting-started-on-the-devops-platform.json:108]:
```suggestion
            "docs/HowTo/RequestGitHubEnterpriseUserLicense.md"
```

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Pull request merged**